packages/uikit/src/hooks/useTooltip/StyledTooltip.tsx

```diff
@@ -1,4 +1,5 @@
 import styled from "styled-components";
+import { m } from "framer-motion";
```

Comment:
Can we rename to frameMotion or something for readability

Reply:
ok